Typo on web clients (#2817)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan-media-bypass.md

### Requirements for using media processors

> Media Processors are always in the media path for voice applications and for Web clients (for example, Teams client in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.

Reviewer: ```suggestion
Media Processors are always in the media path for voice applications and for Web clients (for example, Teams clients in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```

Reviewer 2: commit, then I'll merge.

Media bypass is supported with all Teams endpoints.

> Note for web clients (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even of it started as a bypass call. This happens automatically and does not require any actions on the administrator. 

Reviewer: ```suggestion
Note for web clients (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even if it started as a bypass call. This happens automatically and does not require any actions from the administrator. 
```

Reviewer 2: Commit, then I'll merge
